src/error.ts

```diff
@@ -83,5 +83,5 @@ export function sendError (res: ServerResponse, error: Error | H3Error, debug?:
   }
 
   res.setHeader('Content-Type', MIMES.json)
-  res.end(JSON.stringify(responseBody, null, 2))
```

Comment:
Likewise, we can support the second argument for formatting and pass `debug` flag in app when calling (by default format enabled should be better IMO)

Reply:
Let's leave this one pretty as it's unlikely to make much difference with errors, then.